runelite-client/src/main/java/net/runelite/client/plugins/pyramidplunder/PyramidPlunderConfig.java

```diff
@@ -155,4 +155,20 @@ default int highlightSarcophagusFloor()
 	{
 		return 9;
 	}
+
+	@ConfigItem(
+		position = 11,
+		keyName = "highlightPenultimateUrnsAndUp",
+		name = "Highlight urns from penultimate floor",
+		description = "Highlights the urns from the penultimate floor and up"
+	)
+	default boolean highlightPenultimateUrns() { return false;}
+
+	@ConfigItem(
```

Comment:
There's `@Units` you can use, it might be worth using game ticks here instead to make it eadier to compare with the timer